lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -1013,15 +1234,12 @@ class HybridTextEditing {
   void _startEditing() {
     assert(!isEditing);
     isEditing = true;
-    editingElement.enable(
-      _configuration,
-      onChange: (EditingState editingState) {
-        channel.updateEditingState(_clientId, editingState);
-      },
-      onAction: (String inputAction) {
-        channel.performAction(_clientId, inputAction);
-      }
-    );
+    editingElement.enable(_configuration,
+        onChange: (EditingState editingState) {
+      channel.updateEditingState(_clientId, editingState);
+    }, onAction: (String inputAction) {
+      channel.performAction(_clientId, inputAction);
+    });
```

Comment:
The old formatting was much better. Consider adding a trailing comma so `dartfmt` does a better job.

Reply:
good idea. done. thanks